app/src/main/java/org/schabi/newpipe/util/PlaylistItemsUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.schabi.newpipe.util;
+
+import org.schabi.newpipe.database.playlist.PlaylistLocalItem;
+import org.schabi.newpipe.database.playlist.PlaylistMetadataEntry;
+import org.schabi.newpipe.database.playlist.model.PlaylistRemoteEntity;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public final class PlaylistItemsUtils {
```

Comment:
In general I'm not a fan of creating a util class for just one function. I don't have a better place though.

Reply:
I think a static function in [PlaylistLocalItem](https://github.com/TeamNewPipe/NewPipe/blob/13f66b076d73ea7f126adab4a6c4bb8b213b6ace/app/src/main/java/org/schabi/newpipe/database/playlist/PlaylistLocalItem.java) would be a good solution (that now is a 10 lines file, i.e. an interface, so it makes sense to add another function that takes care of implementers of that interface)